Header header

string[] name
int16[] value